cli-commands/compiler/etherlime-compile/compilerSupplier.js

```diff
@@ -39,7 +39,10 @@ CompilerSupplier.prototype.load = function () {
     const self = this;
     const version = self.config.version;
     const isNative = self.config.version === 'native';
-    const nodeModulesSolc = `${process.cwd()}/node_modules/solc`;
+
+    // We check for current working directory and if contains 'coverageEnv' we return one level up, because when 'etherlime coverage' is running, it makes temporary directory for coverage tests otherwise return just the current working directory
+    const nodeModulesSolc = ((process.cwd().indexOf('coverageEnv')) > -1 ? (path.resolve(process.cwd() + '/../node_modules/solc')) : (process.cwd() + '/node_modules/solc'));
```

Comment:
This row is a bit too complicated and unreadable to me - therefore the need of comment. I'd rewrite this by creating a function isCoverage that returns (process.cwd().includes('coverageEnv')). Also I will create a function getLocalSolcPath as it adds to readability. Lastly I'd use string literals instead of concatenation.